iceberg/pom.xml

```diff
@@ -208,6 +208,11 @@
         <artifactId>junit-jupiter-api</artifactId>
         <version>${junit.jupiter.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
that is already declared in a root pom, we could remove this from iceberg

Reply:
root pom uses the version 5.6.2: https://github.com/apache/hive/blob/6d2340eb4be333406b8c6ffbb7dc67605e41de15/pom.xml#L169
but iceberg pom uses version 5.10.0:
https://github.com/apache/hive/blob/6d2340eb4be333406b8c6ffbb7dc67605e41de15/iceberg/pom.xml#L38

Reply:
oh, could we update the version in root pom?

Reply:
could we try to upgrade global version, so we want have dependency hell
btw I think declaration of `junit-jupiter-api` in not required in iceberg. since it's a. transitive dependency of `junit-jupiter-params`

Reply:
could we try updating global version to avoid dependency hell
also I think we could remove declaration of `junit-jupiter-api` from iceberg pom since it's a transitive dependency of 'junit-jupiter-params'